src/write_lat.c

```diff
@@ -80,6 +80,12 @@ int main(int argc, char *argv[])
 		return FAILURE;
 	}
 
+	/* In case of ib_write_bw, PCI relaxed ordering should be disabled since we're polling for data change
+	 * of last packet so in case of relaxed odering we might get the last packet in wrong order thus the test
+	 * would be incorrect
+	 */
+	user_param.disable_pcir = 1;
```

Comment:
It is still interesting to benchmark write latency with RO enabled, which would be possible if you added a way to use write with immediate and ibv_poll_cq() on the receiver side.
Would you accept such a request?

Reply:
Yes, such a PR will be gladly accepted